pkg/clients/scheduler/client.go

```diff
@@ -55,7 +56,9 @@ func GetSchedulerClient() SchedulerClient {
 
 // Function to get a schedule from the remote scheduler server
 func (schedulerClient *schedulerRestClient) QuerySchedule(id string) (models.Schedule, error) {
-	client := &http.Client{}
+	client := &http.Client{
+		Timeout: time.Second * 10,
```

Comment:
Why this timeouts? As far as I know, every micro service defines its timeouts at startup:
```
http.TimeoutHandler(nil, time.Millisecond*time.Duration(command.Configuration.Service.Timeout), "Request timed out")
```
Why override this setting with another hard coded value?

Reply:
We had a hard coded value in the original code base.  Change can be made to utilize some of our existing timeouts :) . Noted.

Reply:
updated to use the configuration supplied ServiceTimeout.